src/sonic-config-engine/minigraph.py

```diff
@@ -313,8 +319,8 @@ def parse_dpg(dpg, hname):
         for mgmtintf in mgmtintfs.findall(str(QName(ns1, "ManagementIPInterface"))):
             intfname = mgmtintf.find(str(QName(ns, "AttachTo"))).text
             ipprefix = mgmtintf.find(str(QName(ns1, "PrefixStr"))).text
-            mgmtipn = ipaddress.IPNetwork(ipprefix)
-            gwaddr = ipaddress.IPAddress(int(mgmtipn.network) + 1)
+            mgmtipn = ipaddress.ip_network(UNICODE_TYPE(ipprefix), False)
```

Comment:
Thanks for fixing this. It looks like we have test gap and it is not captured during the py3 package build.

Reply:
It's not a fix. I changed from importing the `ipaddr` package `as ipaddress` to importing the actual `ipaddress` package.